### DISCUSSION TITLE
Airbyte Cloud - Platform - Connections - Synch now Button

### DISCUSSION THREAD

**Discussion by KevinConseilxyz** [airbytehq/airbyte#38460] [Connector Ideas and Features]
Hi Airbyte team,

I think it will help, me at least, to be able to synch connections directly from the connection page, cf. screenshot below. Because at the moment i need to go to each connection and click on synch now.

Best,
Kevin

![image](https://github.com/airbytehq/airbyte/assets/54310235/7e5ee8e5-8042-4291-8a04-e5176b631321)

**Comment by NAjustin** [comment]
I agree it would be nice, but I would propose this as a bulk action—e.g. being able to select multiple and either sync, disable, enable, update schema, etc. all of the selected items at once.

From a UI standpoint, I think this could be achieved as a hover state on the existing status icon and a checkbox in the header row to allow select/deselect all (ideally also responsive to the existing filtering). In addition to solving for the bulk sync option, this would also help in cases where, say, you need to temporarily disable all connectors of a certain type or for a certain destination due to an API key issue or migration or such.

Going this direction would allow even more functionality without adding any more clutter to the UI because it's repurposing an existing column. There's plenty of precedent out there in other tools for this shared UI for bulk selection as well.

**Reply by KevinConseilxyz** [reply]
Great and better idea @NAjustin! I like it :)

[a comment by an automated account is not shown]
